README.md

```diff
@@ -58,4 +58,36 @@ SBT depedency:
 
 ````scala
 libraryDependencies += "com.softwaremill.common" %% "id-generator" % "1.0.0"
+````
+
+## Future Try extensions
+
+[![Maven Central](https://maven-badges.herokuapp.com/maven-central/com.softwaremill.common/futuretry_2.11/badge.svg)](https://maven-badges.herokuapp.com/maven-central/com.softwaremill.common/futuretry_2.11)
+
+Provides two utility methods for extending `Future`:
+
+ - `tried: Future[Try[T]]` - reifying the Future's result.
+ - `transTry(f: Try[T] => Try[S]): Future[S]` - corresponds to 2.12's new `transform` variant, allowing to supply a single function (if, for example, you already have one handy), 
```

Comment:
Alright, I'll change it. 

But, for the record, I still disagree with the counter-rationale (I consider CRs to be the second line of defense against this sort of thing, and I don't think that `trans` is that ambiguous ).